### DISCUSSION TITLE
Outstanding ticket

### DISCUSSION THREAD

**Discussion by soubal12** [vercel/community#3372] [Help]
### Summary

Hi, sorry to make this here. I submitted a ticket with level 3 severity, 6 days ago. But still it hasn't been responded to. Can somebody at Vercel help, please? Thank you.

### Steps to Reproduce

See above.

**Comment by amyegan** [comment]
If you share the case number, I can check the status for you

**Reply by soubal12** [reply]
hi @amyegan ticket 00146934 (7 days old) and 00147744 (2 days old). Thank you
